components/NoteModal.js

```diff
@@ -0,0 +1,62 @@
+const T = require('prop-types');
+const { View } = require('react-native');
+const { default: Styled } = require('styled-components');
+const { Text } = require('components/Type');
+const { Input } = require('@ui-kitten/components');
```

Comment:
Tiniest nit— I just find it friendly to have the local project components grouped together separately from external dependencies.
```suggestion
const { Input } = require('@ui-kitten/components');
const { Text } = require('components/Type');
```

Reply:
I agree! I'm usually picky about that too and must have just missed this.